cmd/backups_new.go

```diff
@@ -61,6 +61,15 @@ backup.`,
 			options["compressed"] = false
 		}
 
+		location, err := cmd.Flags().GetString("location")
+		if err == nil && cmd.Flags().Changed("location") {
+			if location == "" {
+				options["location"] = nil
+			} else {
+				options["location"] = location
+			}
+		}
+
```

Comment:
The API asks that you provide `null` as the value for `location` if you want to put the backup in local storage specifically and that isn't your default backup location. That's a bit tricky with the CLI since you can't provide `nil` or `null` so I settled on this. The user can add `--location` or `--location ""` to the command and that becomes `null` in the API call. `""` is not valid as a mount name normally so this doesn't create a conflict, whereas using the actual text `null` or `local` could be a mount name.